assets/airports/rjaa.json

```diff
@@ -0,0 +1,1153 @@
+{
+    "radio": {
+        "twr": "Narita Tower",
+        "app": "Tokyo Approach",
+        "dep": "Tokyo Departure"
+    },
+    "icao": "RJAA",
+    "iata": "NRT",
+    "magnetic_north": 0,
+    "ctr_radius": 80,
+    "ctr_ceiling": 27000,
```

Comment:
Is this the actual ceiling of the approach control's airspace? Or else, is there a good reason for it to be so high?